src/atc/transformers/timezone_transformer.py

```diff
@@ -0,0 +1,62 @@
+import pyspark.sql.functions as F
+from pyspark.sql import DataFrame
+from timezonefinder import TimezoneFinder
+
+from atc.etl.extended_transformer import ExtendedTransformer
+
+
+class TimeZoneTransformer(ExtendedTransformer):
+    """
+    This transformer extracts a timezone using longitude and latitude.
+
+    Attributes:
+    ----------
+        longitude_col : float
+            name of column with longitude values
+        latitude_col : float
+            name of column with latitude values
+        dataset_input_key : str
+            input dataset identifier
+        dataset_output_key : str
+            output dataset identifier
+        column_output_name : str
+            output column name of the TimeZone column
+
+    Methods
+    -------
+    process(df: DataFrame):
+        returns the input DataFrame with a TimeZone column
+    """
+
+    def __init__(
+        self,
+        longitude_col: float,
+        latitude_col: float,
+        dataset_input_key: str,
+        dataset_output_key: str,
```

Comment:
dataset_input_key and data_output_key are optional in the extended transformer. Is it on purpurse that they are required here?